prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -140,67 +163,16 @@ private async Task Send(string from, IEnumerable<string> to, IEnumerable<string>
         }
 
-        private string GetClinicManagerEmailBody(Enrollee enrollee, EnrolmentCertificateAccessToken token)
+        private async Task<string> GetClinicManagerEmailBody(Enrollee enrollee, EnrolmentCertificateAccessToken token)
         {
-            var body = "To: Clinic Manager (person responsible for coordinating PharmaNet access):<br><br>";
-
-            body += $"{enrollee.FirstName} {enrollee.LastName} has been approved for Community Practice Access to PharmaNet.<br><br>";
-
-            body += "<b>To set up their access, you must forward this PRIME Enrolment Confirmation";
-            body += " and the information specified below to your <u>PharmaNet Software Vendor</u>.</b><br><br>";
-
-            body += "<ol>";
-
-            body += "<li style='margin-bottom:0.75rem;'>";
-            body += "Name of Medical Clinic:";
-            body += "</li>";
-
-            body += "<li style='margin-bottom:0.75rem;'>";
-            body += "Clinic Address:";
-            body += "</li>";
-
-            body += "<li style='margin-bottom:0.75rem;'>";
-            body += "Pharmacy Equivalency Code (PEC): <i>(this is your PharmaNet site ID - ask your Vendor, if you are unsure)</i>";
-            body += "</li>";
-
-            body += "<li style='margin-bottom:0.75rem;'>";
-            body += "For <b>Physicians, Pharmacists, and Nurse Practitioners:</b><br><br>";
-            body += "College Name and College ID of this user: ";
-            body += "<i>(leave this blank if this user is not a Physician, Pharmacist or Nurse Practitioner)</i>";
-            body += "</li>";
-
-            body += "<li style='margin-bottom:0.75rem;'>";
-            body += "For users who work <b>On Behalf Of</b> a Physician, Pharmacist, or Nurse Practitioner:<br><br>";
-            body += "College Name(s) and College ID(s) of the Physicians, Pharmacists or Nurse Practitioners ";
-            body += "who this user will access PharmaNet on behalf of: ";
-            body += "<i>(leave this blank if this user is a Pharmacist, Nurse Practitioner or Physician)</i>";
-            body += "</li>";
-
-            body += "</ol>";
-
-            body += $"<a href=\"{token.FrontendUrl}\">{token.FrontendUrl}</a>. ";
-            body += $"<b>This link will expire after {_certificateService.GetMaxViews()} views or {_certificateService.GetExpiryDays()} days.</b><br><br>";
-
-            body += "Thank you,<br><br>";
-
-            body += "PRIME<br><br>";
-            //TODO: Is this stored as an attribute in the backend?
-            body += "1-844-39PRIME<br><br>";
-            //TODO: Is this stored as an attribute in the backend?
-            body += "<a href='mailto:PRIMEsupport@gov.bc.ca' target='_top'>PRIMEsupport@gov.bc.ca</a>";
-
-            return body;
+            EmailProps emailProps = new EmailProps(enrollee.FirstName, enrollee.LastName, token.FrontendUrl, _certificateService.GetMaxViews(), _certificateService.GetExpiryDays());
```

Comment:
These props are created identically in these two methods, can just move them out into the send provisioner link method

Reply:
When you do that, these metods just pass the props into the razor converter, so these methods are a little unnessisary

Reply:
The only difference is there is no provisionerName in one of them